sdk/servicebus/service-bus/src/util/utils.ts

```diff
@@ -180,3 +182,198 @@ export function getAssociatedReceiverName(
   }
   return receiverName;
 }
+
+/**
+ *  @ignore
+ * Helper utility to retrieve `string` value from given input,
+ * or undefined if not passed in.
+ * @param value
+ */
+export function getStringOrUndefined(value: any): string | undefined {
+  if (value == undefined) {
+    return undefined;
+  }
+  return value.toString();
+}
+
+/**
+ *  @ignore
+ * Helper utility to retrieve `integer` value from given string,
+ * or undefined if not passed in.
+ * @param value
+ */
+export function getIntegerOrUndefined(value: any): number | undefined {
+  if (value == undefined) {
+    return undefined;
+  }
+  const result = parseInt(value.toString());
+  return result == NaN ? undefined : result;
+}
+
+/**
+ *  @ignore
+ * Helper utility to retrieve `boolean` value from given string,
+ * or undefined if not passed in.
+ * @param value
+ */
+export function getBooleanOrUndefined(value: any): boolean | undefined {
+  if (value == undefined) {
+    return undefined;
+  }
+  return (
+    value
+      .toString()
+      .trim()
+      .toLowerCase() === "true"
+  );
+}
+
+/**
+ *  @ignore
+ * Helper utility to retrieve `JSON` value from given input,
+ * or undefined if not passed in. Treats empty string as invalid JSON
+ * @param value
+ */
+export function getJSONOrUndefined(value: any): any | undefined {
+  if (value == undefined || (typeof value === "string" && value.trim() == "")) {
+    return undefined;
+  }
+
+  const formattedStringValue = JSON.stringify(value).trim();
```

Comment:
```suggestion
  const formattedStringValue = JSON.stringify(value);
```

Are there cases where JSON.stringify produces extra whitespace?

Reply:
No, this is updated (although, pending conclusion on concern about the final implementation of the helper utility)